### PR TITLE
Little touches here and there

### PR DESCRIPTION
**Pull request by xijo**

### CODE REVIEW

File: spec/taro/rails/integration_spec.rb [hunk 1]
```diff
@@ -32,6 +36,7 @@ def show
 
     get(:show, params: { user: { name: 'taro' } })
 
-    expect(response.body).to eq('TARO')
+    debugger
```
Comment (jaynetics) [inline spec/taro/rails/integration_spec.rb:39]:
```suggestion
```

File: spec/taro/rails/integration_spec.rb [hunk 2]
```diff
@@ -12,14 +12,18 @@
...
     controller_class = Class.new(ActionController::Base) do
       def self.name = 'UsersController'
 
       api 'my api'
       accepts input_type
-      returns ok: String
+      returns ok: response_type
       def show
-        render json: @api_params[:name].upcase
+        render json: { name: @api_params[:name].upcase }
```
[the remark this answers is not in the document]
Comment (jaynetics) [reply spec/taro/rails/integration_spec.rb:26]: more info 

https://github.com/taro-rb/taro/blob/93e38b0e0ec71d78e5001a487d0d1e5173de3853/lib/taro/types/shared/nesting.rb#L1-L5

https://github.com/taro-rb/taro/blob/93e38b0e0ec71d78e5001a487d0d1e5173de3853/README.md?plain=1#L9-L10

[the remark this answers is not in the document]
Comment (xijo) [reply spec/taro/rails/integration_spec.rb:26]: Hmm but the spec should not be passing then right?

[the remark this answers is not in the document]
Comment (jaynetics) [reply spec/taro/rails/integration_spec.rb:28]: true! not sure why it does 🤔

**Pull request merged**